internal/scripts/scripts.go

```diff
@@ -171,10 +175,17 @@ func (c *ContentValue) Write(thread *starlark.Thread, fn *starlark.Builtin, args
 
 	// No mode parameter for now as slices are supposed to list files
 	// explicitly instead.
-	err = os.WriteFile(fpath, fdata, 0644)
+	entry, err := fsutil.Create(&fsutil.CreateOptions{
+		Path: fpath,
+		Data: bytes.NewReader(fdata),
```

Comment:
What happened with the mode?

Reply:
The mode is not used because we are changing an existing file, it had to be there before because `os.WriteFile` has three args but our in our `CreateOptions` we can skip it. I saw your comment https://github.com/canonical/chisel/pull/131#discussion_r1583122911 and I thought it was simpler to remove it altogether.

Reply:
The comment there said the explicit type was irrelevant because 0644 is a constant.

What's the interface being implemented?  What does the comment above Mutated say?

Reply:
https://github.com/canonical/chisel/pull/131#discussion_r1634370497